src/web-components/DisconnectWalletButton/index.ts

```diff
@@ -0,0 +1,81 @@
+import { css, html, LitElement } from 'lit';
+
+declare global {
+    interface Window {
+        cardano: any;
+    }
+}
+import { customElement, property, state } from 'lit/decorators.js';
+import { DisconnectWalletButtonStyles } from './styles';
+
+/**
+ * `disconnect-wallet-button` is a custom component for disconnecting a specified wallet.
+ * 
+ * ### Help: How to Use the `<disconnect-wallet-button>` Component
+ * 
+ * 1. **walletKey (required)**:
+ *    - Pass the wallet's key (string) to identify the wallet to be disconnected.
+ *    - **Example**:
+ *      ```html
+ *      <disconnect-wallet-button walletKey="nami"></disconnect-wallet-button>
+ *      ```
+ * 
+ * 2. **addFunction (optional)**:
+ *    - Pass a function that will execute when the button is clicked. This can be used to handle disconnection logic.
+ *    - **Example**:
+ *      ```html
+ *      <disconnect-wallet-button 
+ *          walletKey="nami" 
+ *          .addFunction=${() => console.log('Wallet disconnected!')}>
+ *      </disconnect-wallet-button>
+ *      ```
+ * 
+ * **Note**: The `walletKey` needs to match the key in `window.cardano` for the wallet you're targeting.
+ */
+
+
+@customElement('disconnect-wallet-button')
+export class DisconnectWalletButton extends LitElement {
+    @property({ type: String }) walletKey!: string;
```

Comment:
instead of passing in walletKey, let's pass in the walletIconUrl. This way we can get rid of the global declaration above